cloudsmith_cli/core/api/files.py

```diff
@@ -59,6 +59,9 @@ def request_file_upload(owner, repo, filepath, md5_checksum=None):
 def upload_file(upload_url, upload_fields, filepath, callback=None):
     """Upload a pre-signed file to Cloudsmith."""
     upload_fields = upload_fields.items()
+    
+    if not hasattr(upload_fields, 'append'):
+        upload_fields = list(upload_fields)
```

Comment:
This works, although we could also take advantage of the `six` module and do something like this:

```
import six
[snip]
upload_fields = list(six.iteritems(upload_fields))
```

Could also leverage the `future` module (although I prefer `six` for compatibility code):

```
import future
[snip]
upload_fields = future.listitems(upload_fields)
```

Thoughts?